.github/scripts/check_icon_pr.py

```diff
@@ -18,10 +18,15 @@ def main():
     try:
         all_icons = filehandler.get_json_file_content(args.devicon_json_path)
 
+        devicon_err_msg = []
+        #First check if devicon.json is sorted
+        if sorted(all_icons, key=lambda d: d['name']) != all_icons:
+            devicon_err_msg.append("devicon.json is not sorted correctly. Please make sure that your icon is added in the `devicon.json` file at the correct alphabetic position as seen [here](https://github.com/devicons/devicon/wiki/Updating-%60devicon.json%60)")
```

Comment:
Restructure the error message for better visualization.
```suggestion
            devicon_err_msg.append(f"devicon.json is not sorted correctly.\nPlease make sure that your icon is added in the `devicon.json` file at the correct alphabetic position\nas seen here: https://github.com/devicons/devicon/wiki/Updating-%60devicon.json%60")
```

### Preview
![preview](https://user-images.githubusercontent.com/76992016/206909391-9670a57d-8a7d-431d-960a-69bf6a7314c9.png)